react/features/recording/actions.js

```diff
@@ -113,20 +113,50 @@ export function showRecordingError(props: Object) {
  *
  * @param {string} streamType - The type of the stream ({@code file} or
  * {@code stream}).
+ * @param {string?} participantName - The participant name stopping the recording.
  * @returns {showNotification}
  */
-export function showStoppedRecordingNotification(streamType: string) {
+export function showStoppedRecordingNotification(streamType: string, participantName?: string) {
     const isLiveStreaming
         = streamType === JitsiMeetJS.constants.recording.mode.STREAM;
+    const descriptionArguments = { name: participantName };
     const dialogProps = isLiveStreaming ? {
-        descriptionKey: 'liveStreaming.off',
+        descriptionKey: participantName ? 'liveStreaming.offBy' : 'liveStreaming.off',
+        descriptionArguments,
         titleKey: 'dialog.liveStreaming'
     } : {
-        descriptionKey: 'recording.off',
+        descriptionKey: participantName ? 'recording.offBy' : 'recording.off',
+        descriptionArguments,
         titleKey: 'dialog.recording'
     };
 
-    return showNotification(dialogProps, NOTIFICATION_TIMEOUT);
+    return showNotification(dialogProps, NOTIFICATION_TIMEOUT * 2);
+}
+
+/**
+ * Signals that a started recording notification should be shown on the
+ * screen for a given period.
+ *
+ * @param {string} streamType - The type of the stream ({@code file} or
+ * {@code stream}).
+ * @param {string} participantName - The participant name that started the recording.
+ * @returns {showNotification}
+ */
+export function showStartedRecordingNotification(streamType: string, participantName: string) {
+    const isLiveStreaming
+        = streamType === JitsiMeetJS.constants.recording.mode.STREAM;
+    const descriptionArguments = { name: participantName };
+    const dialogProps = isLiveStreaming ? {
+        descriptionKey: 'liveStreaming.onBy',
+        descriptionArguments,
+        titleKey: 'dialog.liveStreaming'
+    } : {
+        descriptionKey: 'recording.onBy',
+        descriptionArguments,
+        titleKey: 'dialog.recording'
+    };
+
+    return showNotification(dialogProps, NOTIFICATION_TIMEOUT * 2);
```

Comment:
Is the double timeout intentional? Does it make sense to have a NOTIFICATION_LONG_TIMEOUT? Or maybe we could just modify the constant if it's not used elsewhere?

Reply:
So I had added the same and before for notifying who kicked who. The idea is to give more time of this notification as it contains a name, gives more time to the user to read it before closing it.
Now on second thought with kick there are two names in the notification where here we have one. I can change it as NOTIFICATION_TIMEOUT is 2.5 secs and this will stay for 5 seconds.
WDYT?

Reply:
I'm good with that 👍

Reply:
Updated lib-jitsi-meet and revert it back to default notification time after talking with @yanas.